features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/utils/forge/DDTraceIdForgeryFactory.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.android.utils.forge
+
+import com.datadog.trace.api.DDTraceId
```

Comment:
so now we have 2 API packages: one is from `dd-trace-java` with things like `DDTraceId` and another from us with things like `DatadogTraceId`?